pygmt/datasets/samples.py

```diff
@@ -123,3 +123,26 @@ def load_fractures_compilation():
     fname = which("@fractures_06.txt", download="c")
     data = pd.read_csv(fname, header=None, sep=r"\s+", names=["azimuth", "length"])
     return data[["length", "azimuth"]]
+
+
+def load_hotspots():
```

Comment:
```suggestion
def _load_hotspots():
```
Based on the conversation at https://github.com/GenericMappingTools/pygmt/pull/1439/files#r687250893, I think this should have a leading underscore if it is not meant to be exported and will not be added to the API docs.

Reply:
I think the function will be added to the API docs, because it will be used in gallery examples.

Reply:
Agree with @seisman, this sample hotspot dataset should be public facing, so there should be an entry added to `doc/api/index.rst` at https://github.com/GenericMappingTools/pygmt/blob/v0.4.1/doc/api/index.rst#datasets.

Reply:
added in c146942